baaas-decision-fleet-manager/src/main/java/org/kie/baaas/dfm/app/dao/DecisionDAO.java

```diff
@@ -42,8 +42,9 @@ public Decision findByCustomerAndIdOrName(String customerId, String decisionIdOr
     }
 
     public long getDecisionCountByCustomerId(String customerId) {
-        Parameters params = Parameters.with("customerId", customerId);
```

Comment:
This can be simplified to:

```
return getEntityManager().createNamedQuery("Decision.decisionCountByCustomerId", Long.class).setParameter("customerId", customerId).getSingleResult();
```

Reply:
nice